src/Domain/EcsContainer.php

```diff
@@ -4,31 +4,32 @@
 
 namespace NunoMaduro\PhpInsights\Domain;
 
-use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symplify\EasyCodingStandard\HttpKernel\EasyCodingStandardKernel;
+use Symplify\PackageBuilder\Console\Input\InputDetector;
 
 /**
  * @internal
  */
 final class EcsContainer
 {
     /**
-     * @var \Symfony\Component\DependencyInjection\Container
+     * @var \Symfony\Component\DependencyInjection\ContainerInterface
      */
     private static $container;
 
-    /**
-     * @return \Symfony\Component\DependencyInjection\Container
-     */
-    public static function make(): Container
+
+    public static function make(): ContainerInterface
     {
         if (self::$container === null) {
-            if (file_exists(__DIR__ . '/../../vendor/symplify/easy-coding-standard/bin/container.php')) {
-                $containerPath = __DIR__ . '/../../vendor/symplify/easy-coding-standard/bin/container.php';
-            } else {
-                $containerPath = __DIR__ . '/../../../../symplify/easy-coding-standard/bin/container.php';
+            $easyCodingStandardKernel = new EasyCodingStandardKernel('phpinsights', InputDetector::isDebug());
+            $easyCodingStandardKernel->boot();
+
+            if ($easyCodingStandardKernel->getContainer() === null) {
+                throw new \RuntimeException('Unable to get EcsContainer');
```

Comment:
I think this cannot happen after boot() method. It would fail inside with an exception

Reply:
= it can be removed :)

Reply:
I added this because PHPStan said : 

```
Static property NunoMaduro\PhpInsights\Domain\EcsContainer::$container   
(Symfony\Component\DependencyInjection\ContainerInterface) does not      
accept Symfony\Component\DependencyInjection\ContainerInterface|null.
```

Reply:
Yea, PHPStan is stupid here, now aware of context. Ignore error in `phpstan.neon` with note